README.md

```diff
@@ -1,3 +1,8 @@
 # Design Patterns Study Group
 
 Contains notes and exercises done as part of the Bristol DP study group.
+
+## Add maven project
+In case you open the whole repository in the IntelliJ and it doesn't give you an opportunity to run the project:
```

Comment:
I don't think you need the before IntelliJ, but you could keep it in for dramatic effect!

Reply:
The 'the'? 😅